firestore/app/src/main/java/com/google/example/firestore/kotlin/DocSnippets.kt

```diff
@@ -34,8 +32,6 @@ import java.util.concurrent.TimeUnit
 /**
  * Kotlin version of doc snippets.
  *
- * Note: this is a naive auto-translation of the Java code with a few edits, this is not intended
- *       as a good example of idiomatic Kotlin.
```

Comment:
@samtstern I took the liberty to remove this note, but let me know if you wish to keep it.